llvm/lib/Transforms/Scalar/CorrelatedValuePropagation.cpp

```diff
@@ -47,11 +47,6 @@ using namespace llvm;
 
 #define DEBUG_TYPE "correlated-value-propagation"
 
-static cl::opt<bool> CanonicalizeICmpPredicatesToUnsigned(
```

Comment:
Did something change for the example mentioned in https://reviews.llvm.org/D112895#3149487 to warrant removal of this option?

Reply:
The example is optimized by ConstraintElimination now.